### DISCUSSION TITLE
Radar detector

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#339] [Q&A] [answered]
After a few thousand kilometers in D, F and ES I can say:
These radar devices saved in Open Street Map are probably from the day before yesterday.
You can safely forget that.
A speed camera app like Blitzer.de is better.

Just in case no one has noticed yet....
This is more confusing than it's worth!

**Comment by devemux86** [comment]
We can use data that exist at the exact moment of search in the OpenStreetMap database.

But this will need internet, it can not be done offline like current implementation. :slightly_smiling_face:

**Comment by 0709wiwiwi** [comment]
Wooow Leeb64...The billl will hopefully not run up too high on these speed tests :-) 
It is of course true that osm is volunteer and amateur work.  Amateur in the not so good sense when it comes to the reliability and quality of the saved data. So is unreliable for the serious work. Read the traffic signs ?

**Comment by WolfgangZ42** [comment]
Never used Blitzer.de and don't use the Cruiser implementation. I'm reading the traffic signs and add VAT - that's my personal limit. And I'm paying not more then two tickets per year ... 
Okay - there are countries (Switzerland, Norway, etc.) where I forget the VAT, to expensive.

**Comment by Leeb64** [comment]
The speed camera warning thing is quite useful in countries like France.
At a limit of 80km/h (country road)...in no time you roll along - without meaning to - at a comfortable 86 (5km tolerance) - costs 48 euros.
The whole thing should then work to some extent.
That's why my tip is to take something that works there.
What good are unnecessary warnings?(whether created volunteer or not)
I've had very good experiences with Blitzer.de. It costs 5 euros once.
You may have saved that right away...
Even these speed camera trailers are often reported.

**Comment by devemux86** [comment, marked as answer]
POI search (online) for more up-to-date results will be available in:

- https://github.com/devemux86/cruiser/discussions/375
